pynumenc_meta.py

```diff
@@ -1,13 +1,10 @@
 """Define meta information about numenc package."""
 
 __title__ = 'numenc'
-__description__ = 'Translate integer and floating-point numbers to/from ' \
-                  'sortable bytes'
+__description__ = 'Convert numbers to/from sortable bytes'
 __url__ = 'https://github.com/Parquery/pynumenc'
 __version__ = '1.0.0'
-__author__ = 'Teodoro Filippini'
-__author_email__ = 'teodoro.filippini@parquery.com'
+__author__ = 'Teodoro Filippini, Marko Ristin'
+__author_email__ = 'teodoro.filippini@parquery.com, marko@parquery.com'
```

Comment:
Please put our personal emal addresses (we might continue working on the open source projects even if we leave Parquery). Mine is marko.ristin@gmail.com.